static_src/actions/space_actions.js

```diff
@@ -5,27 +5,41 @@
  */
 
 import AppDispatcher from '../dispatcher.js';
+import cfApi from '../util/cf_api';
 import { spaceActionTypes } from '../constants.js';
+import SpaceStore from '../stores/space_store';
 
 export default {
   fetch(spaceGuid) {
     AppDispatcher.handleViewAction({
       type: spaceActionTypes.SPACE_FETCH,
       spaceGuid
     });
+
+    // TODO error action should also be specified here
+    return cfApi.fetchSpace(spaceGuid).then(this.receivedSpace);
```

Comment:
I just updated the list of guidelines to include this opinion. You can see this in action in the [toggleQuicklook](https://github.com/18F/cg-dashboard/pull/935/files#diff-5700de5f748b06353d227e8f6a17417aR76) action. `toggleQuicklook` is like a fetch action (because the toggle needs to fetch spaces for the org) and when the request returns, the appropriate `success`/`error` action is called based on how the promise resolves.